### PR TITLE
Small assorted fixes

### PR DESCRIPTION
**Pull request by chaosvolt**
Whee, about time I got done drunkenly fumbling with Github.

* Fixed excess separator in item group file.
* Fixed single incorrectly-capitalized item in item group file.
* Fixed oddities with bolter ammotype. For now rolled up hellfire bolts into the same ammotype as 75mm bolts, but this can be re-split once any weapons are added that explicitly require loading only standard or hellfire bolts.
* Small typo fixes. Unsure about best practice for capitalization for item names is, but went with roughly how item names are capitalized in main game. :V
* Also changed the decription of promethium from a single word to a basic desacription based off the Lexicanum entry ( http://wh40k.lexicanum.com/wiki/Promethium ).

I eventually derped and realized the ammotype-related error was strictly due to a non-existent item, not ammotype. Bolt ammotype is crossbow bolts (hence renaming the ammotype to 75bolt), whereas it's the item ID "bolt" that doesn't exist. Doh.

### CODE REVIEW

File: 40K/ammo.json [hunk 1]
```diff
@@ -65,7 +65,7 @@
     "id": "promethium",
     "type": "AMMO",
     "name": "promethium",
-    "description": "promethium",
+    "description": "A highly volatile fuel used in Imperial incendiary weapons, that ignites on exposure to oxygen.",
```
Comment (Wulfle) [inline 40K/ammo.json:68]: A hypergolic fuel ignites when mixed with O2. Promethium has to be ignited but that's a small gripe.

File: 40K/armor.json [hunk 2]
```diff
@@ -19,7 +19,7 @@
     "id": "flack_helmet",
     "type": "ARMOR",
     "category": "armor",
-    "name": "Imperial Flack Helmet",
```
Comment (Wulfle) [inline 40K/armor.json:22]: Missed that. Thank you.

File: 40K/armor.json [hunk 3]
```diff
@@ -42,7 +42,7 @@
     "id": "flack_armor",
     "type": "ARMOR",
     "category": "armor",
-    "name": "Imperial Flack Armor",
```
Comment (Wulfle) [inline 40K/armor.json:45]: And again. It's great to have another set of eyes looking over something!

File: 40K/magazine.json [hunk 4]
```diff
@@ -104,7 +104,7 @@
 	"material" : "steel",
 	"symbol" : "#",
 	"color" : "light_gray",
-	"ammo_type" : "bolt",
+	"ammo_type" : "75bolt",
```
Comment (Wulfle) [inline 40K/magazine.json:107]: Everything looks good. I'm going to push this in a little bit so make any changes that you need to.

**Review by Wulfle — APPROVED** [review]
Very well done. Super quick too. It's nice to have another set of eyes. I know what I intend to do but I'm new so it's always good to have a mentor. Beleve it or not, the forums and you have taught me more about code and it's application than any textbook or class that I've ever done. 

Thank much!

**Comment by chaosvolt** [comment]
Ah, thanks. Regarding the promethium description, I got that from the description on their wiki. Their claim at least is that the term refers both to generic petroleum fuels, and for a more specific fuel used in flamers, with that being (possibly incorrectly?) described as hypergolic.

**Pull request merged**